MDC-103/ObjectiveC/Complete/Shrine/Shrine/LoginViewController.m

```diff
@@ -148,10 +148,12 @@ - (void)viewDidLoad {
                               toTextInputController:self.usernameTextFieldController];
   [MDCTextFieldColorThemer applySemanticColorScheme:colorScheme
                               toTextInputController:self.passwordTextFieldController];
-  [MDCTextButtonColorThemer applySemanticColorScheme:colorScheme
+  [MDCContainedButtonColorThemer applySemanticColorScheme:colorScheme
```

Comment:
Shouldn't this be a text button?

Reply:
The screenshot in the Swift example was a Contained Button so I just wanted them to match.

Reply:
Done